File: Examples/Node.js/Page-Setup/ExtractPageSetupInformation.js
// ExStart:1
var fs = require('fs');
var assert = require('assert');
var StorageApi = require('asposestoragecloud');
var WordsApi = require('asposewordscloud');
var configProps = require('../Config/config.json');
var data_path = '../../../Data/';

var AppSID = configProps.app_sid;
var AppKey = configProps.api_key;
var config = {'appSid':AppSID,'apiKey':AppKey , 'debug' : true};

// Instantiate Aspose Storage API SDK
var storageApi = new StorageApi(config);
// Instantiate Aspose Words API SDK
var wordsApi = new WordsApi(config);

// Set input file name
var name = "SampleWordDocument.docx";
var sectionIndex = 0;

try {
// Upload source file to aspose cloud storage
storageApi.PutCreate(name, versionId=null, storage=null, file= data_path + name , function(responseMessage) {

	assert.equal(responseMessage.status, 'OK');

	// Invoke Aspose.Words Cloud SDK API to get page setup properties by given section index from a word document
	wordsApi.GetSectionPageSetup(name, sectionIndex, null, null, function(responseMessage) {
			assert.equal(responseMessage.status, 'OK');
			var secPageSetup = responseMessage.body.PageSetup
			if(secPageSetup !==null){
				console.log( "PaperSize :" + responseMessage.body.PageSetup.PaperSize + "  Orientation:  " + responseMessage.body.PageSetup.Orientation);
				}
			});
	});

}catch (e) {
  console.log("exception in example");
  console.log(e);
}
//ExEnd:1

File: Examples/Python/Examples/ExtractPageSetupInformation.py
import asposewordscloud
from asposewordscloud.WordsApi import WordsApi
from asposewordscloud.WordsApi import ApiException
from asposewordscloud.models import SectionPageSetupResponse

import asposestoragecloud
from asposestoragecloud.StorageApi import StorageApi
from asposestoragecloud.StorageApi import ResponseMessage


apiKey = "XXXXX" #sepcify App Key
appSid = "XXXXX" #sepcify App SID
apiServer = "http://api.aspose.com/v1.1"
data_folder = "../../data/"

#Instantiate Aspose Storage API SDK
storage_apiClient = asposestoragecloud.ApiClient.ApiClient(apiKey, appSid, True)
storageApi = StorageApi(storage_apiClient)
#Instantiate Aspose Words API SDK
api_client = asposewordscloud.ApiClient.ApiClient(apiKey, appSid, True)
wordsApi = WordsApi(api_client)

#set input file name
filename = "SampleExecuteTemplate.docx"
sectionIndex = 0

#upload file to aspose cloud storage
storageApi.PutCreate(Path=filename, file=data_folder + filename)

try:
    #invoke Aspose.Words Cloud SDK API to get page setup properties by given section index from a word document
    response = wordsApi.GetSectionPageSetup(name=filename, sectionIndex=sectionIndex)

    if response.Status == "OK":
        secPageSetup = response.PageSetup
        if(secPageSetup is not None):
            #display section page setup info
            print "PaperSize :" + response.PageSetup.PaperSize + "  Orientation:  " + response.PageSetup.Orientation

except ApiException as ex:
            print "ApiException:"
            print "Code:" + str(ex.code)
            print "Message:" + ex.message
